source/_integrations/voip.markdown

```diff
@@ -0,0 +1,23 @@
+---
+title: VoIP
+description: Voice over IP
+ha_category:
+  - Voice
+ha_iot_class: Local Push
+ha_release: '2023.5'
+ha_codeowners:
+  - '@balloob'
+  - '@synesthesiam'
+ha_domain: voip
+ha_integration_type: integration
+ha_quality_scale: internal
```

Comment:
It’s not internal

Reply:
According to the integration manifest, it is, so that would mean the integration manifest in Core is also incorrect.

Reply:
What should it be?

Reply:
I guess at this point: None